contracts/main/CurveTricryptoOptimizedWETH.vy

```diff
@@ -1285,21 +1253,28 @@ def _A_gamma() -> uint256[2]:
 @internal
 @view
 def _fee(xp: uint256[N_COINS]) -> uint256:
+
     fee_params: uint256[3] = self._unpack(self.packed_fee_params)
     f: uint256 = MATH.reduction_coefficient(xp, fee_params[2])
+
+    # During parameter ramping, we raise fees and disable admin fee claiming
```

Comment:
Honestly - I don't like raising fees here. I understand it is for safety, but price oracles would be somewhat imprecise when this is the case